src/events.ts

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
Maybe? I don't know what the issue is, but doing this just means we can't ever optimise there